gradlew.bat

```diff
@@ -70,11 +70,11 @@ goto fail
 :execute
 @rem Setup the command line
 
-set CLASSPATH=%APP_HOME%\gradle\wrapper\gradle-wrapper.jar
+set CLASSPATH=
```

Comment:
This seems wrong... Is this what gradle generated? :eyes: Can you double check this please?

Reply:
revert it back